frontend/src/metabase/lib/colors/palette.ts

```diff
@@ -37,6 +37,7 @@ export const colors: ColorPalette = {
   "bg-yellow": "#FFFCF2",
   shadow: "rgba(0,0,0,0.08)",
   border: "#EEECEC",
+  "border-dark": "#B5BBBF",
```

Comment:
I discussed this with design, and they want to align input borders on this color